modules/deepmerge/README.md

```diff
@@ -0,0 +1,99 @@
+# Deepmerge
```

Comment:
Please add a note of attribution including the original license. 

> This `deepmerge` module is a based on the one by [Imperative-Systems-Inc](https://github.com/Imperative-Systems-Inc/terraform-modules/tree/master/deepmerge). Code with in this folder are licensed under the Creative Commons License.

Copy this file into the `deepmerge/` folder.

https://github.com/Imperative-Systems-Inc/terraform-modules/blob/master/LICENSE.md